lib/matplotlib/cm.py

```diff
@@ -29,17 +27,12 @@
 from matplotlib._cm_listed import cmaps as cmaps_listed
 
 
-cmap_d = {}
-
-
-# reverse all the colormaps.
-# reversed colormaps have '_r' appended to the name.
-
-
+@cbook.deprecated("3.2")
```

Comment:
Will this deprecation be shown when using `revcmap()`? If so, I'd not deprecate it. The message would confuse users and the function is private anyway.

Reply:
sure, I removed the decorator and left a comment reminding to remove that function instead.